lib/screens/edit_user_profile_page.dart

```diff
@@ -14,12 +16,80 @@ class EditUserProfilePage extends StatefulWidget {
 }
 
 class _EditUserProfilePageState extends State<EditUserProfilePage> {
+  late final TextEditingController _nameController;
+  late final TextEditingController _emailController;
+
+  // late final TextEditingController _passwordController;
+
+  late final UserNotifier userNotifier;
+
+  @override
+  void initState() {
+    userNotifier = Provider.of<UserNotifier>(context, listen: false);
+    _nameController = TextEditingController();
+    _emailController = TextEditingController();
+    super.initState();
+    // _passwordController = TextEditingController();
+  }
+
   Widget _title() {
-    return Text(
```

Comment:
Let's keep this to be consistent with the app design

Reply:
Keep the title?

Reply:
I put the style back, if that's what you meant